src/SprykerEco/Zed/Unzer/Dependency/UnzerToQuoteClientInterface.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * MIT License
+ * For full license information, please view the LICENSE file that was distributed with this source code.
+ */
+
+namespace SprykerEco\Zed\Unzer\Dependency;
+
+use Generated\Shared\Transfer\QuoteTransfer;
+
+interface UnzerToQuoteClientInterface
+{
+    /**
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    public function getQuote();
```

Comment:
```suggestion
    public function getQuote(): QuoteTransfer;
```

Reply:
The interface methods have the exactly the same signature as the bridged interface.
https://spryker.atlassian.net/wiki/spaces/RFC/pages/13434886/ACCEPTED+RFC+Bridges